vnext/ReactUWP/Views/SliderViewManager.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include <Views/ShadowNodeBase.h>
+#include "SliderViewManager.h"
+
+#include <Utils/ValueUtils.h>
+
+#include <IReactInstance.h>
+
+#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
+
+namespace winrt {
+using ToggleButton = Windows::UI::Xaml::Controls::Primitives::ToggleButton;
+}
+
+namespace react {
+namespace uwp {
+
+class SliderShadowNode : public ShadowNodeBase {
+  using Super = ShadowNodeBase;
+
+ public:
+  SliderShadowNode() = default;
+  void createView() override;
+  void updateProperties(const folly::dynamic &&props) override;
+};
+
+void SliderShadowNode::createView() {
+  Super::createView();
+
+  auto slider = GetView().as<winrt::Slider>();
+  auto wkinstance = GetViewManager()->GetReactInstance();
```

Comment:
You create them but you never use them. please just remove them.
I guess you copied it from other controls, please make change to other control too.

Reply:
the source does use these values.

---
In reply to: [335126768](https://github.com/microsoft/react-native-windows/pull/3400#discussion_r335126768) [](ancestors = 335126768)